app/posts/2021-11-29-ab-test-blue-links.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'A/B test: remove blue contents links from job listing'
+description: Removing the links increased clicks on ‘Get More Information’ links and had little impact on anything else.
+date: 2021-11-29
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+First impressions of our listing page were dominated by a set of blue contents buttons near the top of the listing. This was particularly obvious on mobile devices, where screen space is limited.
+
+## Hypothesis
+
+If we remove the blue contents buttons from the listing page, users will be more likely to engage with meaningful content on the page, and so be more likely to apply for a relevant role.
+
+## Who we tested with
+
+We tested with 623,146 jobseekers between 1st August and 29th November 2021. 50% of users were assigned each variant.
+
+## Findings
+
+If the test were to be repeated in identical conditions:
+- 2.21 +/- 0.05 % of jobseekers would click on a ‘Get More Information’ link to an external site chosen by the publisher if the blue contents buttons were present
+- 2.39 +/- 0.05 % of jobseekers would click on a ‘Get More Information’ link to an external site chosen by the publisher if the blue contents buttons were absent
+There would be no statistically significant difference between the proportions of jobseekers who downloaded a document (8.2%), saved a job (0.17%) or applied for a job (0.16%). It is possible that running the test for several months longer might yield a very small but statistically significant difference, but if so, it would be so small it would be safe to ignore.
```

Comment:
```suggestion

There would be no statistically significant difference between the proportions of jobseekers who downloaded a document (8.2%), saved a job (0.17%) or applied for a job (0.16%). It is possible that running the test for several months longer might yield a very small but statistically significant difference, but if so, it would be so small it would be safe to ignore.
```